luxonis_train/nodes/ocr_decoder.py

```diff
@@ -0,0 +1,171 @@
+"""ResNet backbone.
+
+Source: U{https://github.com/hailo-ai/LPRNet_Pytorch/blob/master/model/LPRNet.py}
+@license: U{PyTorch<https://github.com/hailo-ai/LPRNet_Pytorch?tab=Apache-2.0-1-ov-file#readme>}
+"""
+from typing import Literal
+
+import torch
+import torch.nn as nn
+import torchvision
+from torch import Tensor
+
+from .base_node import BaseNode
+from luxonis_train.utils.types import LabelType
+
+
+class ResBlock(nn.Module):
+    def __init__(self, ch_in, ch_out, stride=1, ks=3, downsample=None, padding=1):
+        super(ResBlock, self).__init__()
+        self.downsample = downsample
+        self.block = nn.Sequential(
+            nn.Conv2d(in_channels=ch_in, out_channels=ch_out, kernel_size=ks, stride=stride, padding=padding),
+            nn.BatchNorm2d(num_features=ch_out),
+            nn.ReLU(),
+            nn.Conv2d(in_channels=ch_out, out_channels=ch_out, kernel_size=ks, stride=1, padding=padding),
+            nn.BatchNorm2d(num_features=ch_out),
+        )
+        self.act = nn.ReLU()
+
+    def forward(self, x):
+        out = self.block(x)
+        if self.downsample is not None:
+            x = self.downsample(x)
+        out += x
+        out = self.act(out)
+        return out
+
+
+class DownSample(nn.Module):
+    def __init__(self, ch_in, ch_out, kernel_size=3, stride=1, padding=0):
+        super(DownSample, self).__init__()
+        self.block = nn.Sequential(
+            nn.Conv2d(in_channels=ch_in, out_channels=ch_out, kernel_size=kernel_size, stride=stride, padding=padding)
+        )
+
+    def forward(self, x):
+        out = self.block(x)
+        return out
+
+
+class OCRDecoderBackbone(BaseNode):
```

Comment:
Instead of `_task_type` argument, add a class attribute `tasks: list[LabelType] = [LabelType.TEXT]`